backend/migrator/migration/prod/2.14/0005##drop_sheet_legacy_field.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE sheet DROP CONSTRAINT sheet_visibility_check;
+ALTER TABLE sheet DROP COLUMN visibility;
+
+ALTER TABLE sheet DROP CONSTRAINT sheet_type_check;
+ALTER TABLE sheet DROP COLUMN type;
+
+ALTER TABLE sheet DROP CONSTRAINT sheet_source_check;
```

Comment:
Maybe needs to drop `DEFAULT 'BYTEBASE'` too.

Reply:
Drop the `source` column now